src/Helpers/Portfolio/PortfolioHelper.js

```diff
@@ -59,3 +59,11 @@ export async function updatePortfolio(portfolioData) {
     window.console.error(error);
   });
 }
+
+export async function removePortfolio(portfolioId) {
+  return userApi.destroy_portfolio(JSON.stringify ({ portfolio_id: portfolioId })).then(() => {
```

Comment:
I am not that familiar with the apiFunctions yet. Do you need to stringify the request body? It is usually done by default.